wandb/sdk/wandb_init.py

```diff
@@ -962,6 +962,7 @@ def init(
     monitor_gym: Optional[bool] = None,
     save_code: Optional[bool] = None,
     id: Optional[str] = None,
+    fork_from: Optional[Union[RunMoment, str]] = None,
```

Comment:
nit: do we want to move to "fork_from" and "resume_from" or do we like "frok" and "resume"?